x/validator-preference/keeper.go

```diff
@@ -0,0 +1,77 @@
+package keeper
+
+import (
+	"fmt"
+
+	"github.com/tendermint/tendermint/libs/log"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/gogo/protobuf/proto"
+	"github.com/osmosis-labs/osmosis/v12/osmoutils"
+	"github.com/osmosis-labs/osmosis/v12/x/validator-preference/types"
+)
+
+type Keeper struct {
+	storeKey            sdk.StoreKey
+	paramSpace          paramtypes.Subspace
+	stakingKeeper       types.StakingInterface
+	bankKeeper          types.BankInterface
+	communityPoolKeeper types.CommunityPoolInterface
```

Comment:
why not use this as distrkeeper but communityPoolKeeper and a separate Interface?

Reply:
hmm this confused me because we use `DistrKeeper interface` (pool-incentives module) somewhere and `CommunityPoolKeeper interface` (incentives module) 

I will change mine to distrKeeper but it'd be nice to have common standard for it